test/test.js

```diff
@@ -12,191 +12,202 @@ chai.use(require('chai-as-promised'));
 
 var users = ['batman', 'superman', 'green_lantern', 'robin', 'aquaman', 'spiderman'];
 
-describe('authentication', function () {
+var testCases = [
+  'normal',
+  'trailing-slash'
+];
 
-  var dbName = 'http://localhost:5984/testdb';
+testCases.forEach(function (testCase) {
 
-  var db;
+  describe('authentication-' + testCase, function () {
 
-  beforeEach(function () {
-    db = new PouchDB(dbName);
-    return db;
-  });
-  afterEach(function () {
-    return db.logout().then(function () {
-      return db.destroy().then(function () {
-        var usersUrl = utils.getUsersUrl(db);
-        return new PouchDB(usersUrl).then(function (usersDb) {
-          // remove the fake users, hopefully we're in the admin party
-          return usersDb.allDocs({
-            include_docs: true,
-            keys: users.map(function (user) {
-              return 'org.couchdb.user:' + user;
-            })
-          }).then(function (res) {
-            var rows = res.rows.filter(function (row) {
-              return row.doc;
-            });
-            var docs = rows.map(function (row) {
-              row.doc._deleted = true;
-              return row.doc;
+    var dbName = testCase === 'normal' ?
+      'http://localhost:5984/testdb' :
+      'http://localhost:5984/testdb/'; // trailing slash
```

Comment:
I think it would be good have multiple slashes (or add a 3rd test case with 2 slashes at the end) as that is what pouchdb is actually doing.